packages/rspeedy/core/src/config/performance/index.ts

```diff
@@ -67,4 +68,25 @@ export interface Performance {
    * ```
    */
   removeConsole?: boolean | ConsoleType[] | undefined
+
+  /**
+   * Whether to print the file sizes after production build.
+   *
+   * {@link Performance.printFileSize}
+   *
+   * @example
+   *
+   * If you don't want to print any information, you can disable it by setting printFileSize to false:
+   *
+   * ```ts
+   * import { defineConfig } from '@lynx-js/rspeedy'
+   *
+   * export default defineConfig({
+   *   performance: {
+   *     printFileSize: false
+   *   },
+   * })
+   * ```
+   */
+  printFileSize?: boolean | PrintFileSize | undefined
```

Comment:
Nitpick: Arrange in alphabetical order (e.g., place `printFileSize` before `removeConsole`).